src/index.scss

```diff
@@ -0,0 +1,9 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+main {
+  display: block;
+}
+
+$feature-flags: (
+  ui-shell: true,
+);
```

Comment:
The feature-flags mapping should come before the style import. See https://www.carbondesignsystem.com/tutorial/react/step-1/#add-ui-shell-sass